FIX An overflow issue in HashingVectorizer (#19035)
scikit-learn/scikit-learn: doc/whats_new/v1.0.rst

    :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
    where 123456 is the *pull request* number, not the issue number.

> 

Reviewer: ```suggestion
```

:mod:`sklearn.feature_extraction`
.................................

> - |Fix| :there will be a negative in indices of class:`feature_extraction.HashingVectorizer`
>   result when mumurhash function return -2**31. :pr:`19035` by :user:`Liu Yu <ly648499246>`.

Reviewer: ```suggestion
- |Fix| Fixed a bug in class:`feature_extraction.HashingVectorizer` where some
  input strings would result in negative indexes in the transformed data.
  :pr:`19035` by :user:`Liu Yu <ly648499246>`.
```

  settings. :pr:`19002` by :user:`Jon Crall <Erotemic>` and
  :user:`Jérémie du Boisberranger <jeremiedbb>`.

> 

Reviewer: ```suggestion
```

- |Fix| :meth:`ElasticNet.fit` no longer modifies `sample_weight` in place.
  :pr:`19055` by `Thomas Fan`_.

> 

Reviewer: ```suggestion
```
